client/my-sites/preview/main.jsx

```diff
@@ -100,7 +100,7 @@ class PreviewMain extends React.Component {
 			{
 				theme_preview: true,
 				iframe: true,
-				'frame-nonce': this.props.site.options.frame_nonce_site_only,
+				'frame-nonce': this.props.site.options.frame_nonce,
```

Comment:
To safely fetch this we can use lodash get
```jsx
const { site } = this.props;
//...
'frame-nonce': get( site, 'options.frame_nonce' );
```

Reply:
Ah it looks like this component is connected, is there any reason not to use `getSiteOption`?

Reply:
So potentially:

```jsx
//...
const { selectedSiteNonce } = this.props;
//...
'frame-nonce': selectedSiteNonce,
//...
const mapState = state => {
	const selectedSiteId = getSelectedSiteId( state );
	return {
		isPreviewable: isSitePreviewable( state, selectedSiteId ),
		site: getSelectedSite( state ),
        selectedSiteNonce: getSiteOption( state, selectedSiteId, 'frame_nonce' ) || '',
		siteId: selectedSiteId,
	};
};
```

Reply:
My only reason was to minimize changes and stick to the intention of the PR - is it better as a followup?

Reply:
This should be pretty minimal, but If you really want to move it to a follow up I won't object.

Reply:
4564383e239c91c54f2613eddaedbcd711647533 👍